polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -582,6 +605,16 @@ $nav-max-width: 360px;
         opacity: 1;
       }
     }
+
+    &:active {
+      #{$se23} & {
+        // stylelint-disable-next-line -- se23
+        .SecondaryActions {
+          // stylelint-disable-next-line -- required for specific styling se23
+          background-color: var(--p-color-bg-app-selected) !important;
```

Comment:
I think this `!important` could be avoided by modifying the code at line 181:

https://github.com/Shopify/polaris/blob/cb7a0615fa5ad42697855d5ce387f3960eba51df/polaris-react/src/components/Navigation/Navigation.scss#L181-L202

To invert the pseudo & class selectors:

```scss
    &:is(:hover, :focus-visible) {
      background: var(--p-color-bg-subdued-hover);
    }

    // stylelint-disable selector-max-class, max-nesting-depth, selector-max-combinators -- se23
    &.ItemInnerWrapper-display-actions-on-hover {
      @media #{$p-breakpoints-md-up} {
        &:is(:hover, :focus-visible) {
          .SecondaryActions {
            background: var(--secondary-actions-on-hover-background-color);

            &::before {
              // stylelint-disable-next-line -- required for fade effect
              background: linear-gradient(
                to right,
                rgba(0, 0, 0, 0),
                var(--p-color-bg-subdued-hover) var(--p-space-8)
              );
            }
          }
        }
        
        &:is(:active) {
          .SecondaryActions {
            background: var(--p-color-bg-app-selected);
          }
        }
      }
      // stylelint-enable
    }
```